.github/ISSUE_TEMPLATE/bug_report.yaml

```diff
@@ -0,0 +1,25 @@
+name: Bug Report
+description: Thanks for taking the time to fill out this bug report!
+
+body:
+
+  - type: textarea
+    id: description
+    attributes:
+      label: Describe the bug
+      description: A clear and concise description of what the bug is. If applicable, add screenshots or logfiles to help explain your problem.
+    validations:
+      required: true
+
+  - type: textarea
+    id: reproduction
+    attributes:
+      label: To Reproduce
+      description: |
+        If applicable describe the steps to and additional information to reproduce the behavior like stand alone usage, usage within the libocpp, usage with a TPM module, happens during a specific libocpp CSMS message, compile options and your system information
```

Comment:
```suggestion
        If applicable describe the steps to and additional information to reproduce the behavior. This could include information about the usage with OCPP, ISO15118 and/or a TPM module.
```